tests/test_flash_triton_torch.py

```diff
@@ -181,8 +182,15 @@ def gen_bias(attn_impl: str):
         tp = torch_name_param_map[n]
         assert p.grad is not None
         assert tp.grad is not None
-        assert allclose_helper(p, tp)
```

Comment:
I this line should not be removed.

Reply:
oh, is this removed because they are initialized to be the same?

Reply:
no.. that was unintentional, thank you for catching it